iris/static/js/student.js

```diff
@@ -1,77 +1,61 @@
-/**
- * Created by s on 22.02.17.
- */
-
-var socket = io();
+const button_slow = document.getElementById("slow");
```

Comment:
To differentiate between normal variables and HTML elements. But I can change it.